app/commands/donations/stripe/sync_subscriptions.rb

```diff
@@ -19,11 +20,22 @@ def create_subscriptions!
     end
   end
 
+  def update_subscriptions!
+    subscription_ids = local_subscription_ids & active_stripe_subscription_ids
+    subscription_ids.each do |subscription_id|
+      subscription = local_subscriptions[subscription_id]
+      stripe_subscription = active_stripe_subscriptions[subscription_id]
+
+      update_subscription_status(subscription, SUBSCRIPTION_STATUSES[stripe_subscription.status])
+    end
+  end
+
   def deactivate_subscriptions!
     inactive_subscription_ids = local_subscription_ids - active_stripe_subscription_ids
     inactive_subscription_ids.each do |subscription_id|
       subscription = local_subscriptions[subscription_id]
-      Donations::Subscription::Deactivate.(subscription) if subscription.active
+
+      update_subscription_status(subscription, SUBSCRIPTION_STATUSES[subscription.status])
```

Comment:
This should use the `stripe_subscription` for the updating bit (same as the update block above does)

```suggestion
      stripe_subscription = Stripe::Subscription.retrieve(subscription_id) # **This line should change to this**

      update_subscription_status(subscription, SUBSCRIPTION_STATUSES[stripe_subscription.status])
```